internal/commands/secrets/secrets_push.go

```diff
@@ -25,6 +26,16 @@ type SecretsPushOptions struct {
 	Force       bool
 }
 
+var secretsPushExample = templates.Examples(`
+	# Push secrets:
+
+    # This will push secrets for "squibby" service
+    ize secrets push squibby
+    
+    # This will push secrets for "squibby" service from a "example-service.json" file to the AWS SSM storage with force option(values will be overwritten if exist)
```

Comment:
```suggestion
    # This will push secrets for "squibby" app from a "example-service.json" file to the AWS SSM storage with force option (values will be overwritten if exist)
```